--- a/plugins/python/uwsgiplugin.py	2016-05-12 17:37:49.000000000 +0000
+++ b/plugins/python/uwsgiplugin.py	2016-06-28 10:41:11.708641508 +0000
@@ -10,7 +10,7 @@
         pass
     return version
 
-NAME='python'
+NAME='python35'
 GCC_LIST = ['python_plugin', 'pyutils', 'pyloader', 'wsgi_handlers', 'wsgi_headers', 'wsgi_subhandler', 'web3_subhandler', 'pump_subhandler', 'gil', 'uwsgi_pymodule', 'profiler', 'symimporter', 'tracebacker', 'raw']
 
 CFLAGS = ['-I' + sysconfig.get_python_inc(), '-I' + sysconfig.get_python_inc(plat_specific=True) ] 
